### ISSUE TITLE
Package in conda-forge

### ISSUE THREAD

**Issue by jeromekelleher** [jeromekelleher/msprime#406]
Currently hoping to get a package for msprime into conda-forge. Full release of 0.5.0 is currently waiting on this.

**Comment by jeromekelleher** [comment]
First need svgwrite: feedstock PR here: https://github.com/conda-forge/staged-recipes/pull/5085

**Comment by jeromekelleher** [comment]
Looks like we have a working conda package on conda-forge, and full support for windows on py35 and py36! Just waiting on this to be merged into conda-forge (which will probably take ~1 week), and we can do some testing then. 

https://github.com/conda-forge/staged-recipes/pull/5140

**Comment by jeromekelleher** [comment]
Done! 

https://github.com/conda-forge/msprime-feedstock
